Makefile

```diff
@@ -15,15 +15,16 @@ style:
 
 package:
 	@git clean -xdf
-	@cp setup_gpl.py setup.py
 	@mkdir -p miss_hit_core/resources/assets
 	@cp docs/style.css miss_hit_core/resources
```

Comment:
So mh_metric can produce HTML report; and the resources needed for that would be missing from the package `pre-commit` would build locally.

Not fatal, as the tool falls back to hit-linking to https://florianschanda.github.io/miss_hit, but a minor annoyance.

Sadly, moving the docs dir into anything not called `docs` would mean major changes to how I get github to host https://florianschanda.github.io/miss_hit

So many moving parts with random requirements :D